src/docs/markdown/quick-starts/https.md

```diff
@@ -7,7 +7,9 @@ title: HTTPS quick-start
 This guide will show you how to get up and running with [fully-managed HTTPS](/docs/automatic-https) in no time.
 
 <aside class="tip">
-	Caddy uses HTTPS for all sites by default, as long as a host name is provided in the config. This tutorial assumes you want to get a publicly-trusted site (i.e. not "localhost") up over HTTPS, so we'll be using a public domain name and external ports.
+	Caddy uses HTTPS for all sites by default, as long as a host name is provided in the config. This tutorial assumes you want to get a publicly-trusted site (i.e. not "localhost") up over HTTPS, so we'll be using a public domain name and external ports. 
+
+	For a "localhost" setup, see examples [here](/docs/automatic-https#examples).
```

Comment:
This is HTML, so the extra line break here won't render (at least, I think).

This is confusing though, since I don't know what a `"localhost" setup` means. A site using `localhost` as the site name? A local dev environment? I think it is ambiguous.

Reply:
I was trying to refer to the earlier mention of [`"localhost"` in that very doc snippet](https://caddyserver.com/docs/quick-starts/https):

> This tutorial assumes you want to get a publicly-trusted site (i.e. not "localhost") up over HTTPS

Perhaps "setup" is not the right word and doesn't work with that prior context I've quoted. I assumed it meant for a "site address"(URI?) that uses `localhost` and would be accessed as such locally. How should I better clarify that?

Reply:
I think that's my question to you -- what is your examples section really about? It seems that doesn't have a clear answer at this point. I don't see any correlation to "localhost" sites and it's more about configuring auto HTTPS in the Caddyfile. Again, I think that would make a good wiki entry, but feels really out of place here.

Reply:
I've still not used Caddy for much yet beyond testing a small website with `Cache-Control` headers. Initially I had issues with the default config from DockerHub in a local environment, and learned to write a small Caddyfile to disable HTTPS. 

Then I later wanted to try HTTPS locally but wasn't sure about the earlier issue where I think it tried to use LetsEncrypt and naturally failed. Found the `local_certs` option which worked (at the time my site address was `:80` and I probably changed to `localhost` at the same time, unaware that also fixed the issue).

Finally, I needed to test a browser behaviour that is meant to only work with HTTPS, and tried to configure for HTTP and HTTPS without the redirect, that's where I got stuck with the docs. The quickstart guide focuses on an actual public server rather than a local dev environment for testing privately on a `localhost` hostname. The intention was to provide some reference/guidance (in lack of quickstart guide) for such a setup, which at the time when I created this PR, I had several misunderstandings of gotchas that I thought was beneficial to inform others about.

> I don't see any correlation to "localhost" sites and it's more about configuring auto HTTPS in the Caddyfile.

New users wanting to use Caddy locally as I've described are probably using Caddyfile config initially(you'd probably know better if that's really the case). The examples all use `localhost` as a site address, and it's about auto HTTPS configs for a reason as both this file and the linked one with examples are about that specific feature.

The snippets are small and may seem obvious to you with your experience in Caddy as the author, but as a new user I stumbled around a lot unsure of what Caddy was wanting me to do configuration wise(eg I didn't realize that `:80` or `localhost:80` would make any difference regarding how certs were handled, I just kept trying different things as I scrambled around the docs trying to make sense of the magic Caddy was doing under the hood and why the behaviour wasn't always seeming consistent(due to the different implicit behaviours based on subtle configuration changes).

So the examples purpose is to save time and confusion for getting what should be fairly simple to setup and support users to [falling into the pit of success](https://blog.codinghorror.com/falling-into-the-pit-of-success/) better.